src/Backend/Teradata/TeradataImportOptions.php

```diff
@@ -8,6 +8,10 @@
 
 class TeradataImportOptions extends ImportOptions
 {
+    public const CSV_ADAPTER_TPT = 1;
```

Comment:
asi bych byl spise pro string constanty, minimalne ve suitach to bude lepe citelne. Ale to je detail

Reply:
změnul sem